### PR TITLE
metdata zip

### PR DESCRIPTION
**Pull request by ChristianBeilschmidt**

### CODE REVIEW

File: services/src/handlers/workflows.rs [hunk 1]
```diff
@@ -38,14 +40,21 @@ where
...
+                    .service(
+                        web::resource("/metadata")
+                            .route(web::get().to(get_workflow_metadata_handler::<C>)),
+                    )
+                    .service(
+                        web::resource("/provenance")
+                            .route(web::get().to(get_workflow_provenance_handler::<C>)),
+                    )
+                    .service(
+                        web::resource("/all_metadata/zip")
```
Comment (michaelmattig) [inline services/src/handlers/workflows.rs:55]: we use camelCase I think

File: services/src/handlers/workflows.rs [hunk 2]
```diff
@@ -241,7 +269,95 @@ async fn get_workflow_provenance_handler<C: Context>(
...
+/// # Example
+///
+/// ```text
+/// GET /workflow/cee25e8c-18a0-5f1b-a504-0bc30de21e06/all_metadata/zip
+/// Authorization: Bearer e9da345c-b1df-464b-901c-0335a0419227
+/// ```
+/// Response:
+/// <zip archive>
+/// ```
+async fn get_workflow_all_metadata_zip_handler<C: Context>(
```
[the remark this answers is not in the document]
Comment (ChristianBeilschmidt) [reply services/src/handlers/workflows.rs:287]: [40cfda3](https://github.com/geo-engine/geoengine/pull/578/commits/40cfda3ace5de2c0d7fdc027be8ad06a6d2d9c88)

File: services/test.http [hunk 3]
```diff
@@ -129,6 +131,12 @@ Content-Type: application/json
 
 ###
 
+GET http://localhost:3030/workflow/{{getVectorWorkflow.response.body.$.id}}/all_metadata/zip
```
Comment (michaelmattig) [inline services/test.http:134]: fix camelcase

Comment (michaelmattig) [inline services/test.http:134]: fix camelcase

Comment (ChristianBeilschmidt) [reply services/test.http]: [1afd633](https://github.com/geo-engine/geoengine/pull/578/commits/1afd6339067923507a1222aae062b7e9133dd27a)

**Review by michaelmattig — APPROVED** [review]

**Comment by michaelmattig** [comment]
bors merge

**Pull request merged**